rust/rope/src/rope.rs

```diff
@@ -208,6 +216,9 @@ pub fn len_utf8_from_first_byte(b: u8) -> usize {
 #[derive(Clone, Copy)]
 pub struct LinesMetric(usize);  // number of lines
 
+/// Measured unit is newline amount
```

Comment:
I think you'll want periods at the end of these lines, otherwise the markdown formatting will make it all flow together.